### PR TITLE
Fix hardcoding changes from 2787

### PR DESCRIPTION
**Pull request by adonispuente**
In PR 2782, I updated the wrong InventoryTable (there are 3). This PR reverts the hard coding of ShowDeleteButton in an inventory table that gets imported, and hard codes it in the one just for this repo. The reset Filter Button is to always be present as per UI/UX. In inventory the Reset Filter button should always be present with this PR.

 To see if it affects another Repo, you can run something like advisor or vulnerability via` npm run start:proxy:beta -- --port=8004` and run inventory with something like `LOCAL_API=vulnerability:8004~https npm run start:proxy`.

### CODE REVIEW

File: src/components/InventoryTable/InventoryTable.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ const InventoryTable = forwardRef(({ // eslint-disable-line react/display-name
                     paginationProps={paginationProps}
                     loaded={loaded}
                     showTagModal={showTagModal}
-                    activeFiltersConfig={{ deleteTitle: 'Reset filters', ...props.activeFiltersConfig, showDeleteButton: true }}
+                    activeFiltersConfig={{ deleteTitle: 'Reset filters', ...props.activeFiltersConfig }}
```
Comment (bastilian) [inline src/components/InventoryTable/InventoryTable.js:204]: Let's add a test for when the InventoryTable has a `activeFiltersConfig` prop set that it pass it along to the EntityTableToolbar.

Comment (adonispuente) [reply src/components/InventoryTable/InventoryTable.js:204]: Just added

**Review by bastilian — APPROVED** [review]
Awesome! Thank you @adonispuente!

**Comment by bastilian** [comment]
@adonispuente Though, please before merging squash and update the commit msg. ;)

**Pull request merged**